apps/website/src/pages/collaborations.tsx

```diff
@@ -219,8 +227,8 @@ const CollaborationItems = forwardRef<
         <Heading
           level={-1}
           className={classes(
-            "alveus-green-800 mb-6 mt-8 border-b-2 border-alveus-green-300/25 pb-2 text-4xl",
-            index === 0 && "sr-only",
+            "alveus-green-800 mb-6 mt-8 scroll-mt-16 border-b-2 border-alveus-green-300/25 pb-2 text-4xl",
```

Comment:
```suggestion
            "text-alveus-green-800 mb-6 mt-8 scroll-mt-16 border-b-2 border-alveus-green-300/25 pb-2 text-4xl",
```
Not your doing, but `alveus-green-800` isn't a valid class, `text-alveus-green-800` is